shm/inc/Cargo.hpp

```diff
@@ -2,17 +2,21 @@
 
 #include <string>
 
+constexpr size_t MAX_AMOUNT_OF_CARGO{ 1000 };
+
+//class resposible for manage commodity in game
 class Cargo {
 public:
-    Cargo(const std::string&, size_t, size_t);
+    //constructs an object of cargo in game
```

Comment:
There is no needed to explain that C'tor will initialize sth :) Everybody know what C'tor do